.github/workflows/main.yml

```diff
@@ -3,13 +3,9 @@ name: Docker Image CI
 on:
   push:
     branches: [ main ]
-  pull_request:
-    branches: [ main ]
 
 jobs:
-
   build:
-
     runs-on: ubuntu-latest
     if: contains(github.ref == 'refs/heads/production' || github.ref == 'refs/heads/master')
```

Comment:
I would remove this as its kind of redundant and breaking the action
```suggestion
```

Reply:
You're also not even using "master" or "production" right now which is going to confuse GitHub actions a lot